reflex/app.py

```diff
@@ -145,6 +145,13 @@ def __init__(self, *args, **kwargs):
                         Also, if there are multiple client subclasses of rx.State(Subclasses of rx.State should consist
                         of the DefaultState and the client app state).
         """
+        if "state" in kwargs:
```

Comment:
can you move it back under the `if not is_testing_env`? we don't want a bunch of our tests to hit this warning